torch_geometric/contrib/flag/flag.py

```diff
@@ -34,7 +44,10 @@ def forward(
         #   `perturb` argument
         # forward = lambda perturb : model(x+perturb, edge_index)[train_idx]
         # model_forward = (model, forward)
-        training_labels = y_true.squeeze(1)[train_idx]
+
+        # TODO
+        # Tyler, did you mean to have .squeeze(1)??
+        training_labels = y_true[train_idx]
```

Comment:
@tnichols89 please double check me here -- I was getting an error that `training_labels` shape didn't match `out`  from the model when we had `training_labels = y_true.squeeze(1)[train_idx]`

Reply:
This was adapted directly from a [method in the original repo](https://github.com/devnkong/FLAG/blob/main/deep_gcns_torch/examples/ogb/ogbn_arxiv/main.py#L56-L64) so my motivation for adding it was limited to that alone.

Since it's a tensor of the ground truth labels, it's almost certainly of shape `(N, 1)` and I suspect they just added the `.squeeze(1)` to convert it to a vector because PyTorch was complaining about the shape of it.

The code from the original repo that this is based on was committed on Oct 19, 2020, at which point PyTorch 1.7.0 hadn't released yet so the original code is from a pretty old version of torch. It seems plausible that this issue popped up in testing because PyTorch has been changed to expect a vector here rather than an `(N, 1)` tensor since this code was originally written.

I expect this is an inconsequential change that had to be made because we're running on a much newer version of PyTorch.

Reply:
Okay, we can leave it this way for now and then fix it if we run into issues running it on real datasets.